src/test/java/com/stripe/functional/IdempotentTest.java

```diff
@@ -1,68 +0,0 @@
-package com.stripe.functional;
```

Comment:
@ob-stripe We need to look into moving those tests to a different level to make sure we are actively verifying this works but it's the wrong way to test idempotency

Reply:
bumping this just in case